model/src/main/java/org/cbioportal/genome_nexus/model/my_variant_info_model/AlleleCount.java

```diff
@@ -0,0 +1,134 @@
+package org.cbioportal.genome_nexus.model.my_variant_info_model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class AlleleCount
+{
+
+    // allele count in total
+    @Field(value = "ac")
+    private Integer ac;
```

Comment:
Ideally we should name the variables like `alleleCount` instead of `ac`, `alleleCountAfrican` instead of `acAfr` etc. In that case wouldn't even need to add any comments since the variable names would be self descriptive.